src/Razor/test/RazorLanguage.Test/IntegrationTests/ComponentDeclarationIntegrationTest.cs

```diff
@@ -130,7 +130,7 @@ protected override void BuildRenderTree(Microsoft.AspNetCore.Components.Renderin
         #pragma warning restore 1998
 #nullable restore
 #line 1 ""x:\dir\subdir\Test\TestComponent.cshtml""
-            
+
```

Comment:
@Pilchie This is what broke the test.

Reply:
Silly vscode fixing whitespace :)  Thanks @NTaylorMullen

Reply:
```suggestion
            
```